README.md

```diff
@@ -1 +1,131 @@
+    - Click on the `Add` Button to add the URL with the Portnumber and EmailID in the respective fields.(Tooltips helps you with the default Port numbers)
+    - The default PortNumber for a Windows Server is `3389`, where as for a Linux server it is `22`.
+    - The extension polls the remote server for every `5` min by default(click [here](#settings) to customize the interval) and notifies you when the website is down.
+    - An Email is sent to the EmailId mentioned in the respective field.
+    
+    **Checkout the following gif to see the functionality graphically.**
+
+<p align="center">
+  <kbd>
+    <img src="https://raw.githubusercontent.com/ShekharReddy4/Pinger/develop/PingerExtension/src/images/gifs/Websitegif.gif"/>
+  </kbd>
+</p>
+
+#### RemoteDB
+- **Adding a `Remote DataBase` to monitor the status**
+    - Click on the Pinger Icon in the chrome menu to use the extension.
+    - By default the `Website` tab is selected, so head to the `Remote DB` tab.
+    - Click on the `Add` Button to add the URL with the Portnumber and Email to notify.(Tooltips helps you with the default Port numbers)
+    - The default PortNumber for the SqlServer is `1433`, for Oracle it is `1521`.
+    - The extension polls the remote server for every `5` min by default(click [here](#settings) to customize the interval) and notifies you when the RemoteDB is down.
+    - An email is sent to the EmailId mentioned in the respective field.
+    
+    **Checkout the following gif to see the functionality graphically.**
+
+<p align="center">
+  <kbd>
+    <img src="https://raw.githubusercontent.com/ShekharReddy4/Pinger/develop/PingerExtension/src/images/gifs/remotedbgif.gif"/>
+  </kbd>
+</p>
+
+
+#### Settings
+- **To Change/Reset the `Settings` of Pinger**
+    - Click on the Pinger Icon in the chrome menu to use the extension.
+    - By default the `Website` tab is selected, so head to the `Settings` tab.
+    - Add Global Email Address
+        - **Global Email address** is used as a default notifying emailID for all the URL's across all tabs.
+        - Fill in with the emailID in that respective field.
+        
+        - **Run Interval In Minutes** field is used as time interval to poll the servers.
+        - Use the navigation button to increase/decrease *(or)* Fill in with custom number of minutes. By default it is `5` mins and the number **cannot be less than 5**.
+        
+        **Checkout the following gif to see the functionality graphically.**
+
+<p align="center">
+  <kbd>
+    <img src="https://raw.githubusercontent.com/ShekharReddy4/Pinger/develop/PingerExtension/src/images/gifs/SettingsGif.gif"/>
+  </kbd>
+</p>
+
+
+## Contribute                                         
+Contributions are always welcome!
+Please read the contribution guidelines [here](contributing.md).
+
+## License
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)
+
+[**MIT**](LICENSE) © **Kishore Ithadi**
```

Comment:
status table, background process data, enable notifications data, example in desc,

Reply:
about localhost